.github/workflows/nightfalldlp.yml

```diff
@@ -0,0 +1,18 @@
+name: nightfalldlp
+on: [pull_request]
+jobs:
+  nightfalldlp:
+    name: nightfalldlp
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo Action
+        uses: actions/checkout@v2
+
+      - name: nightfallDLP action step
+        uses: nightfallai/nightfall_dlp_action@CreateAction
+        env:
```

Comment:
Is there a difference between using env vars vs. inputs?

Reply:
the differences are minor - the way I'm thinking about it semantically is that `inputs` are like dynamic CLI args you would expect a user to pass in to change the functionality of a program (e.g. `--debug=true`), whereas `env` vars are just static environment variables that are required to run the program (e.g. the `nightfall_api_key`), but don't have much say on functionality. So it seems a little more straightforward to me (if i'm using our product and configuring a Github Workflow) to just pass in everything we need via ENV vars.

The other reason is that in our entrypoint shell script, passing them in as `inputs` mean we can only refer to them by their argument index (e.g. `$1` for arg1, `$2` for second, etc.) whereas we can refrence ENV vars by name, making the entrypoint script easier to reason through and more robust for future dev (by preventing someone from accidentally breaking everything via updating the order of `inputs` in our action file).